src/backend/joanie/core/api/client/__init__.py

```diff
@@ -541,6 +542,50 @@ def withdraw(self, request, pk=None):  # pylint: disable=no-self-use, invalid-na
 
         return Response(status=HTTPStatus.NO_CONTENT)
 
+    @action(detail=True, methods=["POST"])
+    def pay_failed_installment_payment(self, request, pk=None):  # pylint: disable=no-self-use, invalid-name, unused-argument
```

Comment:
I'd rather call this endpoint `submit_installment_payment` WDYT ?

Reply:
it's clearer, let's go for that. Thanks